examples/tv-app/tv-common/include/CHIPProjectAppConfig.h

```diff
@@ -52,9 +52,9 @@
 #define CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT 16
 
 // For casting, we need to allow more ACL entries, and more complex entries
```

Comment:
It sounds like this could benefit from a comment as well. The current comment just says "allow more" ... but allow "more than what number?" also should note that too big of a number may not fit. How does one figure out if big enough or not?

Reply:
I am talking about the number with @chrisdecenzo, let me ask him to answer it

Reply:
See ContentAppPlatform::ManageClientAccess for how these are used.
- Each casting client uses one entry. So by limiting to 8 entries, we are limiting the TV to 8 simultaneous clients.
- Each casting client with no app has at least 8 targets, with 1 app has 9 targets, X apps would be 8+X. So the target value should be larger than 8.
- Each casting client has a single subject, so max subjects per entry could be smaller (default).

I think a better solution is to increase the storage for these (I didn't notice the error). Have you looked at that approach?

Reply:
I am worried about the increase in memory usage for some low-end platforms.

Reply:
I saw your PR about this change, so should I wait for that?
https://github.com/project-chip/connectedhomeip/pull/16331#pullrequestreview-912543334